python/lsst/ts/mthexapod/config_schema.py

```diff
@@ -127,6 +127,15 @@
           Use temperature LUT to compensate the deformation caused by
           temperature gradients or not.
         type: boolean
+      high_current_threshold:
+        description: >-
+          High current threshold in Ampere.
+        type: number
+      no_movement_timeout:
```

Comment:
One, post approval comment. I am not sure "timeout" is the best term to use here. Usually timeout means something "failed" to occur in a specified time frame. But, not moving the hexapod is not really a failure. I would suggest changing this to something like `no_movement_idle_time` or something like that.